site/content/docs/excalidraw.md

```diff
@@ -0,0 +1,26 @@
+# Excalidraw support in flowershow
+
+Flowershow supports displaying your excalidraw images on your site.
+
+## Auto generating Excalidraw images
+
+In Obsidian you can use the excalidraw plugin to auto generate images next to it's relevant file.
+
+_Assuming that you have this plugin installed ..._
```

Comment:
I would use a callout for this. I'd also make this a section saying: install the obsidian excalidraw plugin [linked] if you haven't already.